File: Java/set2/Q7.java
// 7. Write a program to find the grade of a student of a subject based on given table. 

class Q7
{
    public static void main(String[]args)
    {
        int mrk=71;
        String grade=(100>=mrk && mrk>=71)?"Distinction":(mrk<71 && mrk>=61)?"1st division":
                    (mrk<61 && mrk>=51)?"2nd division":(mrk<51 && mrk>=41)?"3rd division":
                    (mrk<41 && mrk>=36)?"pass":(mrk<36 && mrk>=0)?"fail":"invalid number";
        System.out.println("Grade: "+grade);
    }
}

File: Python/set2/Q7.py
# 7. Write a program to find the area of triangle when three points are given.  

import math

x1,y1=5,7
x2,y2=9,2 
x3,y3=2,0

ab=math.sqrt((math.pow((x1-x2),2))+(math.pow((y1-y2),2)))
bc=ab=math.sqrt((math.pow((x2-x3),2))+(math.pow((y2-y3),2)))
ca=ab=math.sqrt((math.pow((x3-x1),2))+(math.pow((y3-y1),2)))

s=(ab+bc+ca)/2
a=math.sqrt(s*(s-ab)*(s-bc)*(s-ca))
print("Area:",a)